projects/safemoon/index.js

```diff
@@ -10,24 +11,40 @@ const sdk = require('../../sdk');
   The protocol holds onto the LP tokens
 */
 
-const SAFE_MOON = '0x8076c74c5e3f5852037f31ff0093eeb8c8add8d3';
 const SAFE_MOON_LP = '0x9adc6fb78cefa07e13e9294f150c1e8c1dd566c0';
+const LOCKER = '0xc8b839b9226965caf1d9fc1551588aaf553a7be6';
 
 async function getLPBalance(block) {
-  const lpBalance = (await sdk.bsc.abi.call({
+  return (await sdk.bsc.abi.call({
     target: SAFE_MOON_LP,
     abi: "bep20:balanceOf",
     block, 
-    params: SAFE_MOON
+    params: LOCKER
   })).output;
+}
 
-  return lpBalance;
+async function getTotalLPSupply(block) {
+  return (await sdk.bsc.abi.call({
+    target: SAFE_MOON_LP,
+    abi: "bep20:totalSupply",
+    block
```

Comment:
should reference block with `block.bsc` (for v2 adapters the block is an object { eth: 1234567, bsc: 6785432 })

Reply:
fixed in https://github.com/opentvl/opentvl/commit/c73841985c33dd9241da1af2a1b0df15a3fcde66